src/fileTransformers/engineType.ts

```diff
@@ -0,0 +1,26 @@
+import { FileTransformer } from '../fileTransformer/fileTransformer'
+
+export const engineType: FileTransformer = (params) => {
+  const { file, parameters } = params
+
+  let content = file.content
+
+  if (file.path === 'prisma/schema.prisma') {
+    if (parameters.engineType) {
+      content = content.replace(
+        `provider = "prisma-client-js"`,
+        `provider = "prisma-client-js" \n engineType = "${parameters.engineType}"`
+      )
+    }
+  }
+  if (file.path === 'package.json') {
+    if (parameters.engineType) {
+      content = content.replace(
+        `"@prisma/client": "3.1.1"`,
+        `"@prisma/client": "3.1.0-integration-data-proxy-engine.1"`
```

Comment:
Does this mean on every version of Prisma, we'll have to update the version? 🤔

Reply:
IIUC the idea was to use a dist-tag to minimize churn here. This is a pre-release, not a dist tag, though.

Reply:
Everyone seems on board with the idea, I pinged that thread again today, I just need someone to actually do it

Reply:
Okay there's a `dataproxy` tag now, @eemmiillyy you can use that in a new PR)

Reply:
https://github.com/prisma/templates-node/pull/37